TASVideos/Pages/Games/Versions/Edit.cshtml.cs

```diff
@@ -174,7 +174,7 @@ public async Task<IActionResult> OnPost()
 
 		return string.IsNullOrWhiteSpace(HttpContext.Request.ReturnUrl())
 			? RedirectToPage("List", new { gameId = GameId })
-			: BaseReturnUrlRedirect($"?GameId={GameId}&GameVersionId={version.Id}");
+			: BaseReturnUrlRedirect($"&GameVersionId={version.Id}");
```

Comment:
this can't be right, because if the game already existed and we selected it without creating, its id won't appear after `?` on catalog page, we now just have `&`, so creating a version redirects to System Error